kafka-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/kafka/KafkaProperties.java

```diff
@@ -460,6 +473,11 @@ public void setEventProcessorMode(EventProcessorMode eventProcessorMode) {
      */
     public static class Fetcher {
 
+        /**
+         * Enables autoconfiguration for consuming events from Kafka
```

Comment:
Minor change suggestion in the description. Especially stating the default may be helpful to end-users.
```suggestion
         * Enables auto-configuration for consuming events from Kafka. Defaults to {@code true}.
```

Reply:
Applied this suggestion